### DISCUSSION TITLE
`getLogger` isn't being emitted when `picologging` is used instead of `logging`

### DISCUSSION THREAD

**Discussion by winstxnhdw** [litestar-org/litestar#3673] [Potential Issues]
Not sure if this should be an issue of `Litestar` or `picologging`, but since Litestar explicitly says that it supports `picologging`, I thought I should leave this here first.

I have a simple exception handler. When `picologging` is used instead of `logging`, it no longer emits the exception.
```python
from picologging import getLogger
# from logging import getLogger

def exception_handler(_, exception: Exception) -> Response[dict[str, str]]:
    """
    Summary
    -------
    the Litestar exception handler

    Parameters
    ----------
    request (Request) : the request
    exception (Exception) : the exception
    """
    getLogger('custom.access').error('Application Exception', exc_info=exception)

    return Response(
        content={'detail': 'Internal Server Error'},
        status_code=HTTP_500_INTERNAL_SERVER_ERROR,
    )
```

I wonder if it has anything to do with any of these limitations: https://microsoft.github.io/picologging/limitations.html.

**Comment by provinzkraut** [comment]
Since it doesn't seem like you're using any logging provided by Litestar, I'm going to say that's a picologging issue.

Do you have any logging configured in Litestar?

**Reply by winstxnhdw** [reply]
Nope. Just using the default. I believe Litestar wrote in the docs that zero config is necessary: https://docs.litestar.dev/2/usage/logging.html

**Reply by provinzkraut** [reply]
That part of the documentation is about how *Litestar* uses picologging. In the example you have provided, you are simply using picologging directly. Litestar doesn't interfere with that in any way.
